test/component/uk/gov/hmrc/organisationsdetailsapi/connectors/IfConnectorSpec.scala

```diff
@@ -67,6 +67,8 @@ class IfConnectorSpec
     .bindings(bindModules: _*)
     .configure(
       "cache.enabled"  -> false,
+      "auditing.enabled"                                       -> false,
+      "auditing.traceRequests"                                 -> false,
       "microservice.services.integration-framework.host" -> "localhost",
```

Comment:
In order to comply with the latest auth upgrade please change the host to `127.0.0.1`. This means that it routes through auth correctly as localhost will bypass.